packages/rich-text/src/plugins/EmbeddedEntityBlock/index.tsx

```diff
@@ -45,14 +47,27 @@ function getWithEmbeddedEntityEvents(
     };
 }
 
+function withLinkRenderedTracking(tracking: TrackingProvider) {
+  return function LinkedEntityBlockWithTracking(props: ComponentProps<typeof LinkedEntityBlock>) {
+    return (
+      <LinkedEntityBlock
+        onEntityFetchComplete={() => {
+          tracking.onViewportAction('linkRendered');
```

Comment:
I noticed that as well but I did not find a place where that info is read upon. In the web app we only count how many 'linkRendered' actions are emitted against the number of links in the field. Once the numbers match the tracking data are sent and don't contain any data from the single actions.

Reply:
Let's ping @DanweDE to double-check. As for me, it looks fine

Reply:
Yes, I don't think we're using it. What is `key` even in this context? `${entityType}:${entityId}`? It would make sense to send this info eventually as it would help us to address this TODO [here](https://github.com/contentful/user_interface/blob/7be464e71f2996773a78116220700cfbf6341591/src/javascripts/app/entity_editor/LoadEventTracker.js#L119) but not necessary right now.